### PR TITLE
Deploy to prod when we merge to main.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
Just run `bin/deploy-prod` - the ubuntu-18 environment runs it just fine, which is great and lucky for us.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: .github/workflows/docker_build.yaml [hunk 1]
```diff
@@ -19,3 +19,21 @@ jobs:
...
+    steps:
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+
+      - name: Checkout
+        if: contains(toJSON(steps.file_changes.outputs.files), 'infra/')
+        uses: actions/checkout@v2
+
+      - name: Run deploy-prod.
+        if: contains(toJSON(steps.file_changes.outputs.files), 'infra/')
```
Comment (hmhsu) [inline .github/workflows/docker_build.yaml:34]: **Suggestion** (non-blocking): Might worth deploying even if we don't change the stack, i.e. push a new image?

**Review by hmhsu — APPROVED** [review]

File: .github/workflows/docker_build.yaml (hunk 1 above)
Comment (ndmckinley) [reply .github/workflows/docker_build.yaml:34]: That's a good idea, but I think we want a much less invasive change there - just want to update the container.  Let me see what I can do there...

Comment (ndmckinley) [reply .github/workflows/docker_build.yaml]: Great, got it.

File: .github/workflows/push.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+jobs:
+  build_container:
+    runs-on: ubuntu-18.04 # for older version of awscli.
+    steps:
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+
+      - name: Checkout
+        if: contains(toJSON(steps.file_changes.outputs.files), 'infra/')
```
Comment (hmhsu) [inline .github/workflows/push.yaml:15]: **Question**: Do we now still need this condition? We checkout no matter the change is in `infra/` or not.

**Review by hmhsu — APPROVED** [review]
Looks good. Thanks!

File: .github/workflows/push.yaml (hunk 2 above)
Comment (ndmckinley) [reply .github/workflows/push.yaml:15]: Hm, thanks!  You're right.

**Comment by ndmckinley** [comment]
Okay, here we goooo

**Pull request merged**